engine/installation/linux/SUSE.md

```diff
@@ -1,112 +1,301 @@
+
+## Install Docker
+
+You can install Docker in different ways, depending on your needs:
+
+- Most users
+  [set up Docker's repositories](#install-using-the-repository) and install
+  from them, for ease of installation and upgrade tasks. This is the
+  recommended approach.
+
+- Some users download the RPM package and install it manually and manage
+  upgrades completely manually.
+
+- Some users cannot use third-party repositories, and must rely on the version
+  of Docker in the OpenSUSE or SLES repositories. This version of Docker may be
+  out of date. Those users should consult the CentOS documentation and not
+  follow these procedures.
+
+### Install using the repository
+
+Before you install Docker for the first time on a new host machine, you need to
+set up the Docker repository. Afterward, you can install, update, or downgrade
+Docker from the repository.
+
+#### Set up the repository
+
+1.  Verify and import Docker's public key, which is used to sign packages in
+    Docker's repository.
+
+    First, verify that the fingerprint is `58118E89F3A912897C070ADBF76221572C52609D`:
+
+    ```bash
+    $ curl -s https://yum.dockerproject.org/gpg | gpg --quiet --with-fingerprint
+
+    pub  4096R/2C52609D 2015-07-14
+          Key fingerprint = 5811 8E89 F3A9 1289 7C07  0ADB F762 2157 2C52 609D
+    uid                            Docker Release Tool (releasedocker) <docker@docker.com>
+    ```
+
+    If the fingerprint matches, import the key:
+
+    ```bash
+    $ sudo rpm --import https://yum.dockerproject.org/gpg
+    ```
+
+2.  Use the following command to set up the **stable** repository:
+
+    ```bash
+    $ sudo zypper addrepo \
+        https://yum.dockerproject.org/repo/main/opensuse/13.2/ \
+        docker-main
+    ```
+
+3.  **Optional**: Enable the **testing** repository. You can enable it alongside
+    the stable repository. Do not use unstable repositories on on production
+    systems or for non-testing workloads.
+
+    > **Warning**: If you have both stable and unstable repositories enabled,
+    > updating without specifying a version in the `zypper install` or
+    > `zypper update` command will always install the highest possible version,
+    > which will almost certainly be an unstable one.
+
+
+    ```bash
+    $ sudo zypper addrepo \
+        https://yum.dockerproject.org/repo/testing/opensuse/13.2/ \
+        docker-testing
+    ```
+
+    You can disable a repository at any time by running the `zypper rmrepo`
+    command. The following command disables the `testing` repository.
+
+    ```bash
+    $ sudo zypper removerepo docker-testing
+    ```
+
+#### Install Docker
+
+1.  Update the `zypper` package index.
+
+    ```bash
+    $ sudo zypper refresh
+    ```
+
+2.  Install the latest version of Docker, or go to the next step to install a
+    specific version.
+
+    ```bash
+    $ sudo zypper install docker-engine
+    ```
+
+    > **Warning**: If you have both stable and unstable repositories enabled,
```

Comment:
Yes once while setting up the repos and once when actually installing. It's pretty important.